development.dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3
+
+WORKDIR /usr/src/application/
+RUN apt-get install libevent-dev
+COPY . .
+RUN pip install -r requirements.txt
+
+CMD ["/bin/bash", "-c", "flask deploy-script && python main.py"]
```

Comment:
Actually not sure what this is doing. I thought this would throw an error. Mind explaining what it does?

Reply:
sorry you are right, was from a flask template i had and forgot about, i going to fix it